utils/generate-bazel-build-rules.js

```diff
@@ -81,7 +81,9 @@ const generateBazelBuildRules /*: GenerateBazelBuildRules */ = async ({
         items.forEach(item => {
           if (!dependencies.map(d => `"${d}"`).includes(item)) {
             const [, path, name] = item.match(/\/\/(.+?):([^"]+)/) || [];
-            if (projects.includes(path) && basename(path) === name) {
+            // force include target allows for projects to be included as bazel deps even if they are not in package.json deps
+            // it is an edge case that should be avoided if at all possible
+            if (projects.includes(path) && name !== 'force-include') {
```

Comment:
do we need this? IIRC it already retains manual edits without this

Reply:
This is where that logic lives. The problem was it was dependent on how we named the targets, which has now changed to default to library.